src/App.test.jsx

```diff
@@ -2,6 +2,11 @@ import { render, screen } from "@testing-library/react";
 import App from "./App";
 
 test("renders learn react link", () => {
+  const mockGeolocation = {
```

Comment:
No need mock here, you can mock global.navigator.geolocation in the `jest.setup.js`
Something like that
<img width="937" alt="Screenshot 2023-02-20 at 15 32 52" src="https://user-images.githubusercontent.com/14818162/220109147-5724bb19-5818-4938-b2ae-87196e3d631b.png">